src/main/java/com/kiwigrid/helm/maven/plugin/UploadMojo.java

```diff
@@ -73,34 +86,40 @@ protected void uploadSingle(String file) throws IOException, BadUploadException,
 		case NEXUS:
 			connection = getConnectionForUploadToNexus(fileToUpload);
 			break;
+        case HARBOR:
+			uploadToHarbor(fileToUpload);
+			break;
```

Comment:
Alternatively, `return` and have no need for the very big conditional block `if (uploadRepo.getType() != RepoType.HARBOR)`